docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "YAML as a source for building a site"
+---
+
+# Table of Contents
+
+- [Introduction](#Introduction)
+- [Setup](#Setup)
+- [Adding Content](#Adding-some-content)
+- [Gatsby Config](#Gatsby-configuration)
+- [Gatsby Template](#Gatsby-Template)
+- [Joining the pieces](#Joining-the-pieces)
+- [Sourcing from JSON](https://www.gatsbyjs.org/docs/using-gatsby-without-graphql/)
+
+## Introduction
+
+As you're presented with Gatsby, you'll see that you have at your disposal a myriad of ways to pull data into your website, ranging from the most used ones like [Contentful](https://www.contentful.com/), or [WordPress](https://wordpress.com/), [Drupal](https://www.drupal.org/), to the most edgier ones like [ButterCMS](https://buttercms.com/) or [GraphCMS](https://graphcms.com/).
+
+Each one using the correspondent plugin.
+
+This tutorial will help you return to the origins.
+
+Instead of pulling data using any of the available plugins for each provider mentioned, you'll create a simple fully functional Gatsby website that is sourced from a YAML file.
```

Comment:
```suggestion
Instead of pulling data using any of the available plugins for each provider mentioned, you'll create a fully functional Gatsby website sourced from a YAML file.
```

Reply:
Changed the wording to yours